partition/roles/ipmi-catcher/defaults/main.yaml

```diff
@@ -0,0 +1,4 @@
+---
+ipmi_catcher_image_name: metalstack/ipmi-catcher
+ipmi_catcher_image_tag: v0.1.3
```

Comment:
For our own images we do not default tags anymore because if we do people will not explicitly define them anymore and we have to update this role all the time (and then something unexpected can happen in people's deployments).

You can require the image version like this: https://github.com/Donimax/metal-roles/blob/mini-lab-role-migration/control-plane/roles/metal/tasks/main.yaml#L2-L9